docs/proposals/proposal-002-run.md

```diff
@@ -0,0 +1,313 @@
+  - Describe how the pipeline should _run_ the benchmark tests through GitHub Actions for a specific project e.g. Falco
+  - Communicate the changes needed to be made by the Falco team to change the benchmark test to a GitHub Action file.
+  - Provide _modularity_ for the benchmark tests.
+
+### Non-Goals
+
+<!--
+What is out of scope for this proposal? Listing non-goals helps to focus discussion
+and make progress.
+
+It is important to remember that non-goals are still equally important things
+which will be dealt with one day but are not things which need to be dealt with immediately
+within the scope of this work. This helps make sure everyone is crystal clear on the outcomes.
+-->
+
+* Defining or designing the content of benchmark tests themselves, or assigning responsibility for who should write them.
+
+### Linked Docs
+
+<!--
+Provide links to previous discussions, Slack threads, motivation issues or any other document
+with context. It is really helpful to provide a "source of truth" for the work
+so that people aren't searching all over the place for lost context.
+-->
+
+* [Slack discussion on benchmark test framework](https://cloud-native.slack.com/archives/C060EDHN431/p1708416918423089?thread_ts=1708348336.259699&cid=C060EDHN431)
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is:
+outlining your solution to the problem described in the Motivation section.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories
+
+<!--
+Detail the things that people will be able to do if this proposal is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+**CNCF project maintainer creates a new benchmark test for their project**
+
+As a project maintainer, I create and run a benchmark test from a separate GitHub repository or from the green-reviews-tooling repository, by following the steps indicated in the Green Reviews documentation, so that the project’s benchmark tests runs in the Green Reviews pipeline and so that I can see the project's sustainability metrics.
+
+
+**Green Reviews maintainer helps to create a new benchmark test for a specific CNCF project**
+
+As a Green Reviews maintainer, I can help a CNCF project maintainers to define the Functional Unit of a project so that the project maintainers can create a benchmark test.
+
+**Project maintainer modifies or removes a benchmark test**
```

Comment:
CNCF project maintainer?

Reply:
Yes, so the CNCF project maintainer would be the developers of Falco for example. It's not clear?

Reply:
```suggestion
**CNCF Project maintainer modifies or removes a benchmark test**
```